src/WireMock.Net/Matchers/XPathMatcher.cs

```diff
@@ -37,12 +44,20 @@ public XPathMatcher(params AnyOf<string, StringPattern>[] patterns) : this(Match
     /// </summary>
     /// <param name="matchBehaviour">The match behaviour.</param>
     /// <param name="matchOperator">The <see cref="Matchers.MatchOperator"/> to use. (default = "Or")</param>
+    /// <param name="xmlNamespaceMap">The xml namespaces of the xml document.</param>
     /// <param name="patterns">The patterns.</param>
     public XPathMatcher(
         MatchBehaviour matchBehaviour,
         MatchOperator matchOperator = MatchOperator.Or,
+        XmlNamespace[]? xmlNamespaceMap = null,
         params AnyOf<string, StringPattern>[] patterns)
     {
+        _xmlNamespaceMap = new List<XmlNamespace>();
+        if (xmlNamespaceMap != null)
+        {
+            _xmlNamespaceMap.AddRange(xmlNamespaceMap);

```

Comment:
Only when xmlNamespaceMap != null -->
```
_xmlNamespaceMap = xmlNamespaceMap.ToList();
```

Reply:
Is this a problem? But I can change this ;-)

Reply:
Change to nullable array -> done